src/modules/data/soundData.c

```diff
@@ -3,18 +3,16 @@
 #include "core/util.h"
 #include "core/ref.h"
 #include "lib/stb/stb_vorbis.h"
+#include "lib/miniaudio/miniaudio.h"
+#include "audio/audio_internal.h"
```

Comment:
Okay, sure I can do that (inverting relationship so audio depends on audio instead of other way around).

Heh I understand the sentinment, but yeah the ringbuffer in miniaudio is amazing and it'd be silly not to use it imho :S it's even threadsafe so one could have a SoundData in a background thread and just feed it data!!